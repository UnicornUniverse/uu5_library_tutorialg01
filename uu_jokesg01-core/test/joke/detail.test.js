import UuJokesCore from "uu_jokesg01-core";
import { shallow } from "uu5g05-test";

describe(`UuJokesCore.Joke.JokeDetail`, () => {
  it(`default props`, () => {
    const wrapper = shallow(<UuJokesCore.Joke.Detail />);
    expect(wrapper).toMatchSnapshot();
  });
});
