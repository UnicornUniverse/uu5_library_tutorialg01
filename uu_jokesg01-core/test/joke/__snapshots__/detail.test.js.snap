// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UuJokesCore.Joke.JokeDetail default props 1`] = `
<div>
  Visual Component 
  UuJokesCore.Joke.Detail
   says Hello World!
</div>
`;
